src/components/ProfitAndLossDetailReport/profit_and_loss_detail_report.scss

```
@@ -0,0 +1,111 @@
+:root {
+  --pnl-detail-date-col-width: 12%;
+  --pnl-detail-type-col-width: 12%;
+  --pnl-detail-account-col-width: 18%;
+  --pnl-detail-description-col-width: 28%;
+  --pnl-detail-amount-col-width: 15%;
+  --pnl-detail-balance-col-width: 15%;
+}
+
+.Layer__UI__Table__ProfitAndLossDetailReport {
+  table-layout: fixed;
+  width: 100%;
+
+  .Layer__UI__Table-Column__ProfitAndLossDetailReport {
+    &--Date {
+      width: var(--pnl-detail-date-col-width);
+      min-width: 7rem;
+      color: var(--color-base-600);
+    }
+
+    &--Type {
+      width: var(--pnl-detail-type-col-width);
+    }
+
+    &--Account {
+      width: var(--pnl-detail-account-col-width);
+      min-width: 10rem;
+    }
+
+    &--Description {
+      width: var(--pnl-detail-description-col-width);
+      min-width: 12rem;
+    }
+
+    &--Amount {
+      width: var(--pnl-detail-amount-col-width);
+      min-width: 7.5rem;
+      text-align: right;
+    }
+
+    &--Balance {
+      width: var(--pnl-detail-balance-col-width);
+      min-width: 7.5rem;
+      text-align: right;
+    }
+  }
+
+  .Layer__UI__Table-Cell__ProfitAndLossDetailReport {
+    &--Account {
+      color: var(--color-base-600);
+    }
+
+    &--Type {
+      font-size: var(--text-md);
+
+      button {
+        font-size: inherit;
+        font-family: inherit;
+        font-weight: inherit;
+        font-feature-settings: inherit;
+      }
+    }
+
+    &--Amount {
+      text-align: right;
+      color: var(--color-base-800);
+    }
+
+    &--Balance {
+      text-align: right;
+      color: var(--color-base-800);
+    }
+  }
+}
+
+.Layer__profit-and-loss-detail-report__total-row {
+  display: grid;
+  grid-template-columns:
+    var(--pnl-detail-date-col-width)
+    var(--pnl-detail-type-col-width)
+    var(--pnl-detail-account-col-width)
+    var(--pnl-detail-description-col-width)
+    var(--pnl-detail-amount-col-width)
+    var(--pnl-detail-balance-col-width);
+  align-items: center;
+  padding-top: var(--spacing-sm);
+  padding-bottom: var(--spacing-sm);
+  border-top: 1px solid var(--color-base-300);
+}
+
+.Layer__profit-and-loss-detail-report__total-label {
+  grid-column: 1;
+  font-weight: 540;
+  font-size: var(--text-md);
+  line-height: 140%;
+  text-align: left;
+  padding-left: var(--spacing-md);
+  color: var(--color-base-800);
+}
+
+.Layer__profit-and-loss-detail-report__total-amount {
+  grid-column: 5;
+  font-weight: 580;
+  font-size: var(--text-md);
+  line-height: 140%;
+  text-align: right;
+  color: var(--color-base-900);
+  padding-right: var(--spacing-sm);
+}
```

With my recommended changes above, you only need roughly this much CSS.

```suggestion
.Layer__profit-and-loss-detail-report__total-row {
  display: grid;
  grid-template-columns:
    var(--pnl-detail-date-col-width)
    var(--pnl-detail-type-col-width)
    var(--pnl-detail-account-col-width)
    var(--pnl-detail-description-col-width)
    var(--pnl-detail-amount-col-width)
    var(--pnl-detail-balance-col-width);
  border-top: 1px solid var(--color-base-300);
}

.Layer__profit-and-loss-detail-report__total-label {
  grid-column: 1;
  padding-left: var(--spacing-md);
}

.Layer__profit-and-loss-detail-report__total-amount {
  grid-column: 5;
  text-align: right;
  padding-right: var(--spacing-sm);
}
```